examples/lightrag_nvidia_demo.py

```
@@ -0,0 +1,159 @@
+import os
+import asyncio
+from lightrag import LightRAG, QueryParam
+from lightrag.llm import openai_complete_if_cache, nvidia_openai_embedding, nvidia_openai_complete
```

Ruff (0.8.0)

`lightrag.llm.nvidia_openai_complete` imported but unused

Remove unused import: `lightrag.llm.nvidia_openai_complete`

(F401)